.github/workflows/ogf_workflow.yml

```diff
@@ -0,0 +1,395 @@
+              --tags "created on= $(date)" "Use=OGFTest" > /dev/null
+            sleep 3 # once the RG is created
+            
+            az storage account create --resource-group $RESOURCE_GROUP_NAME \
+                --name $STORAGE_ACCOUNT_NAME --location ${{ matrix.region }} \
+                --tags "created on= $(date)" "Use=OGFTest" --sku Standard_LRS
+            sleep 3 # Adding some breathing space for Storage Account creation.
+      - name: Create new Consumption Azure Function App
+        if: ${{ matrix.sku == 'cons' }}
+        uses: Azure/cli@1.0.4
+        with:
+          azcliversion: 2.16.0
+          inlineScript: |
+            az functionapp create --consumption-plan-location ${{ matrix.region }} \
+              --name $FUNCTION_APP_NAME --os-type $OS \
+              --functions-version 3 --runtime $LANGUAGE \
+              --resource-group $RESOURCE_GROUP_NAME \
+              --runtime-version $LANGUAGE_VERSION \
+              --storage-account $STORAGE_ACCOUNT_NAME \
+              --tags "created on= $(date)" "Use=OGFTest"
+            sleep 5 # Adding some breathing space for Storage Account creation.
+
+      - name: Create new EP1 Azure Function App
+        if: ${{ matrix.sku == 'ep1' }}
+        run: |
+          az functionapp plan create --resource-group $RESOURCE_GROUP_NAME \
+            --name $FUNCTION_APP_PLAN_NAME --is-linux true --max-burst 1 \
+            --min-instances 1 --location ${{ matrix.region }} --sku EP1 \
+            --tags "created on= $(date)" "Use=OGFTest"
+          sleep 2
+
+          az functionapp create --resource-group $RESOURCE_GROUP_NAME --plan $FUNCTION_APP_PLAN_NAME \
+            --name $FUNCTION_APP_NAME --functions-version 3 --runtime $LANGUAGE \
+            --runtime-version $LANGUAGE_VERSION --storage-account $STORAGE_ACCOUNT_NAME \
+            --tags "created on= $(date)" "Use=OGFTest"
+          sleep 2
+
+      - name: Create new AppService plan Azure Function App
+        if: ${{ matrix.sku == 'p1v2' }}
+        run: |
+          az functionapp plan create --resource-group $RESOURCE_GROUP_NAME \
+            --name $FUNCTION_APP_PLAN_NAME --is-linux true --number-of-workers 1 \
+            --location ${{ matrix.region }} --sku P1v2 \
+            --tags "created on= $(date)" "Use=OGFTest"
+          sleep 3
+
+          az functionapp create --resource-group $RESOURCE_GROUP_NAME --plan $FUNCTION_APP_PLAN_NAME \
+            --name $FUNCTION_APP_NAME --functions-version 3 --runtime $LANGUAGE \
+            --runtime-version $LANGUAGE_VERSION --storage-account $STORAGE_ACCOUNT_NAME \
+            --tags "created on= $(date)" "Use=OGFTest"
+          sleep 3
+
+      - name: Deploy Tests
+        shell: bash -l {0}
+        run: |
+          az webapp config appsettings set --name $FUNCTION_APP_NAME \
+            --resource-group $RESOURCE_GROUP_NAME \
+            --settings ENABLE_ORYX_BUILD=true $THREAD_COUNT=4
```

Comment:
Should also add a setting call SCM_DO_BUILD_DURING_DEPLOYMENT=1 to enable remote build.